airflow/operators/python.py

```diff
@@ -276,6 +280,17 @@ def task(
     :type multiple_outputs: bool
 
     """
+
+    if python_callable:
+        sig = signature(python_callable).return_annotation
+        ttype = getattr(sig, "__origin__", None)
+
+        if (
+            sig != inspect.Signature.empty
+            and is_container(ttype)
+        ):
```

Comment:
Not sure about using this parenthesis here. Does this not fit in a single line?

Reply:
Nah that's just left over noise, originally had complex check before I found the is_container func existed. I'll clean it up